contracts/Sup_Dao.sol

```diff
@@ -0,0 +1,112 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "hardhat/console.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@unioncredit/v1-sdk/contracts/BaseUnionMember.sol";
+
+/*
+
+*/
+
+contract SupDao is ERC721, BaseUnionMember {
```

Comment:
I'm getting an error here that I'm not sure about.

`(union) sarah@Sarahs-MacBook-Pro Sup_Dao % npx hardhat compile
TypeError: Contract "SupDao" should be marked as abstract.
  --> contracts/Sup_Dao.sol:15:1:
   |
15 | contract SupDao is ERC721, BaseUnionMember {
   | ^ (Relevant source part starts here and spans across multiple lines).
Note: Missing implementation: 
  --> @unioncredit/v1-sdk/contracts/BaseUnionMember.sol:26:5:
   |
26 |     constructor(address _marketRegistry, address _unionToken, address _underlyingToken) {
   |     ^ (Relevant source part starts here and spans across multiple lines).
Note: Missing implementation: 
  --> @openzeppelin/contracts/token/ERC721/ERC721.sol:44:5:
   |
44 |     constructor(string memory name_, string memory symbol_) {
   |     ^ (Relevant source part starts here and spans across multiple lines).


Error HH600: Compilation failed

`